serializers/src/glTF/2.0/babylon.glTFMaterial.ts

```diff
@@ -62,11 +62,25 @@ module BABYLON.GLTF2 {
          */
         private static _maxSpecularPower = 1024;
 
+
+        private static _localEngine: Engine;

```

Comment:
Move this to the exporter as member variables and then either have a parent pointer to the exporter or pass down these values to the material from the exporter.